Modules/TOF/include/Base/Counter.h

```diff
@@ -101,15 +101,15 @@ void Counter<size, labels>::Count(const unsigned int& v)
     LOG(FATAL) << "Incrementing counter too far! " << v << "/" << size;
   }
 #ifdef ENABLE_COUNTER_DEBUG_MODE
-  LOG(INFO) << "Incrementing " << v << "/" << size << " to " << counter[v];
+  LOG(DEBUG) << "Incrementing " << v << "/" << size << " to " << counter[v];
```

Comment:
This is not needed, it's on only if the `ENABLE_COUNTER_DEBUG_MODE` is enabled, please change it back